inc/classes/class-imagify-auto-optimization.php

```diff
@@ -94,8 +117,11 @@ public function remove_hooks() {
 			remove_filter( 'big_image_size_threshold',             [ $this, 'prevent_auto_optimization_when_generating_thumbnails' ], $prio );
 			remove_filter( 'wp_generate_attachment_metadata',      [ $this, 'allow_auto_optimization_when_generating_thumbnails' ], $prio );
 			remove_action( 'imagify_after_auto_optimization_init', [ $this, 'do_auto_optimization' ], $prio );
+			// Upload failure handling.
+			remove_action( 'wp_ajax_media_create_image_subsizes',  [ $this, 'prevent_auto_optimization_when_recovering_from_upload_failure' ], 5 );
```

Comment:
Priority here is `5`, while on the `add_action` it was `-5`

Reply:
Nice catch.

Reply:
It has been fixed.